static/css/around-map.css

```diff
@@ -116,6 +116,8 @@ footer {
   background: rgba(0, 0, 0, 0.6);
   z-index: 2;
   overflow: scroll;
+  z-index: 10;
```

Comment:
10 to be stronger than the header wich need a 9 to be stronger than the map (thank you OSM ^^)